cmd/lncli/walletrpc_active.go

```diff
@@ -606,7 +606,12 @@ var fundPsbtCommand = cli.Command{
 	The optional 'inputs' flag decodes a JSON list of UTXO outpoints as
 	returned by the listunspent command for example:
 
-	    --inputs='["<txid1>:<output-index1>","<txid2>:<output-index2>",...]'
+	    --inputs='["<txid1>:<output-index1>","<txid2>:<output-index2>",...]
+
+	The optional '--change-type' flag permits to choose the address type
+	for the change for default accounts and single imported public keys.
+	The custom address type can only be p2tr at the moment (p2wkh will be
```

Comment:
nit: In the description you use small letters for the address type, but in the description of the `change_type` you use capital letters, so I would recommend doing it uniformly.